keyboards/handwired/dactyl_manuform/4x5/keymaps/kombo/keymap.c

```diff
@@ -0,0 +1,211 @@
+	|--------+--------+--------+--------+--------|                   |--------+--------+--------+--------+--------|
+	|   a    |   o    |   e    |   u    |   i    |                   |   d    |   h    |   t    |   n    |   s    |
+	|--------+--------+--------+--------+--------|                   |--------+--------+--------+--------+--------|
+	|   '    |   q    |   j    |   k    |   x    |                   |   b    |   m    |   w    |   v    |   z    |
+	`--------+--------+--------+-----------------'                   `-----------------+--------+--------+--------'
+	         |   [    |   {    |                                                       |   }    |   ]    |
+	         `-----------------+-----------------.                   ,--------+--------+-----------------'
+	                           |  TAB   | SPACE  |                   |   BS   |  ESC   |
+	                           `--------+--------+--------. ,--------+--------+--------'
+	                                    |   (    |  CTRL  | | SHIFT  |   )    |
+	                                    |--------+--------| |--------+--------|
+	                                    |LEFT ALT|LEFT GUI| |   \    |  RALT  |
+	                                    `-----------------' `-----------------'
+	*/
+	LAYOUT(                                                                                                        \
+	 KC_SCLN, KC_COMM, KC_DOT,  KC_P,    KC_Y,                        KC_F,    KC_G,    KC_C,    KC_R,    KC_L,    \
+	 KC_A,    KC_O,    KC_E,    KC_U,    KC_I,                        KC_D,    KC_H,    KC_T,    KC_N,    KC_S,    \
+	 KC_QUOT, KC_Q,    KC_J,    KC_K,    KC_X,                        KC_B,    KC_M,    KC_W,    KC_V,    KC_Z,    \
+	          KC_LBRC, KC_LCBR,                                                         KC_RCBR, KC_RBRC,          \
+	                   KC_TAB,  KC_SPC,                                        KC_BSPC, KC_ESC,                    \
+	                            KC_LPRN, KC_LCTL,                     KC_LSFT, KC_RPRN,                            \
+	                                     KC_LALT, KC_LGUI,   KC_BSLS, KC_RALT                                      \
+	)
+};
+
+bool
+process_record_user(uint16_t keycode, keyrecord_t *record) {
+	uint8_t static n_keys = 0;
+	record->event.pressed ? n_keys++ : n_keys--;
+	if (n_keys >= 8) {
+		kombo_enable();
+		n_keys = 0;
+		clear_keyboard();
+	}
+
+        switch (keycode) {
+		case SRST:
+			if (record->event.pressed) {
+				matrix_init_user();
+			}
+			return false;
+	}
+
+	return true;
+}
+
+bool
+process_kombo_event(size_t kombo_index, keypos_t pos) {
+	uint16_t const keycode = key_kombos[kombo_index].keycode;
+
+	if (keycode == KC_CCOFF) {
+		kombo_disable();
+		return false;
+	}
+
+	if (IS_MOD(keycode)) {
+		set_oneshot_mods(MOD_BIT((uint8_t)keycode) | get_oneshot_mods());
+		return false;
+	}
+
+	return true;
+}
+
+void matrix_init_user(void) {
+	clear_keyboard();
```

Comment:
I don't think that this is actually needed here, as I believe that this is called on startup, anyways.